app/main/helpers.js

```diff
@@ -12,3 +12,23 @@ export function setSavedEnv () {
   const set = makeSetSavedEnv(settings);
   return set();
 }
+
+export function getAppiumSessionFilePath (argv, isPackaged, isDev) {
+  if (process.platform === 'darwin' && !isDev) {
+    // packaged macOS apps do not receive args from process.argv, they
+    // receive the filepath argument from the `electron.app.on('open-file', cb)` event
+    return false;
+  }
+  let argvIndex = isPackaged ? 1 : 2;
+  return argv[argvIndex];
+}
+
+// get the slice of the redux state that's needed for the .appiumsession files
+export function getSaveableState (reduxState) {
```

Comment:
why do we care about just getting the slice? is it because it's serialized and passed as a message or something?

Reply:
@jlipps good question. Do you think it would make more sense to just save the entire state?

I was actually thinking a potential use-case would be to save recorded actions too.

Reply:
i dunno, the whole state might not make sense. i was just trying to make sure i understood what you were doing. i think we can always add more state in later versions of save files